src/KafkaFlow.Compressor/ConfigurationBuilderExtensions.cs

```diff
@@ -33,6 +34,7 @@ public static IConsumerMiddlewareConfigurationBuilder AddCompressor<T>(this ICon
 
         /// <summary>
         /// Registers a middleware to compress the message
+        /// Is highly recommended to use the native compression ('WithCompression()' method) instead of using the compressor middleware
```

Comment:
It would be better to use the `<see cref="xxxxxxxxxxx"/>` instead of just the method name

Reply:
This can't be done because _WithCompressor_ is an extension method created in the main projetct (KafkaFlow), so Kafkaflow.Compressor would need to reference the main project (KafkaFlow) and this wouldn't make sense in our architecture.